### PR TITLE
Add cluster-api-provider-azure v1.13 milestone to plugins

### PR DESCRIPTION
**Pull request by willie-yao**
/assign @CecileRobertMichon @jackfrancis

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -517,9 +517,9 @@ milestone_applier:
     release-0.7: v0.7.x
     release-0.6: v0.6.x
   kubernetes-sigs/cluster-api-provider-azure:
-    main: v1.12
-    release-1.11: v1.11
-    release-1.10: v1.10
+    main: v1.13
+    release-1.11: v1.12
```
Comment (jackfrancis) [inline config/prow/plugins.yaml:521]: both the key and val need to be updated, e.g.:

`release-1.12: v1.12`

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**